docs/user/QUICKSTART.md

```diff
@@ -38,10 +38,11 @@ Create the HTTPRoute to route traffic through Envoy proxy to the example app:
 kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/v0.2.0-rc1/examples/kubernetes/httproute.yaml
 ```
 
-### Testing the configuration
+### Test the Configuration
+#### Option 1: Use Kubectl Port Forwarding
 Port forward to the Envoy service:
 ```shell
-kubectl -n envoy-gateway-system port-forward service/envoy-eg 8888:8080 &
+kubectl -n envoy-gateway-system port-forward service/envoy 8888:8080 &
```

Comment:
this is not really true, since the image points to v0.2.0 which creates `envoy-eg`

Reply:
Thanks for catching this. I removed the changes to the quickstart guide since it's pinned to rc1. I've added a note to https://github.com/envoyproxy/gateway/issues/318 so we can think about having a quickstart based on `main`.